index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Mint Sign-up Clone</title>
+  </head>
+
+  <body>
+    <nav>
+      <div class="sign-in">
+        <div class="sign-txt">
+          <p>Already have an Intuit account?</p>
+          <a href="#">I forgot my User ID or Password</a>
+        </div>
+        <div class="signin-btn">
+          <button><span class="custom-span"></span>Sign In</button>
+        </div>
+      </div>
+    </nav>
+    <div class="title">
+      <a href="#"><img src="intuit.png" alt="intuit"/></a>
+    </div>
+      <div class="product-row">
+
+        <ul>
+        <li><img src="mint.svg" alt="mint icon"/> mint</li>
+        <li><img src="quickbooks.svg" alt="quickbooks icon"/> quickbooks</li>
+        <li><img src="turbotax.svg" alt="turbotax icon"/> turbotax</li>
+	</ul>
+      </div>
+      
+    <div class="container">
+      <div class="form-container">
+        <div class="form-header">
+          <h2>Create an Inuit Account</h2>
+          <p>
+            One account for everything Intuit, including Mint.
+          </p>
+          <span><a href="#"> Learn more.</a></span>
+        </div>
+        <form action="/do-something" method="post">
+          <div class="input-box">
+            <label for="email"> Email address </label>
+            <input type="email" name="email" id="email" required />
+          </div>
+
+          <div class="input-box">
+            <label for="phone"> Phone (recommended) </label>
+            <input type="tel" name="phone" id="phone" required />
+            <p class="p-charges">
+              Standard call, messaging or data rates may apply.
+            </p>
+          </div>
+
+          <div class="input-box">
+            <label for="password"> Password <span></span></label>
+            <input type="password" name="password" id="password" required />
+          </div>
+          <div class="input-button">
+            <button name="create"><span class="custom-span"></span> Create Account</button>
```

Comment:
please change this to ```<input type='submit' value='Create Account>```. this was already pointed out by the previous reviewer. make only this one change and this project will get approved.